src/SoulMemory/EldenRing/EldenRing.cs

```diff
@@ -173,7 +175,17 @@ private ResultErr<RefreshError> InitPointers()
                 _igt.Clear();
                 return Result.Err(new RefreshError(RefreshErrorReason.UnknownException, "soulmods injection failed"));
             }
-            
+
+            if (_process?.GetModuleExports(_process.Is64Bit().Unwrap() ? "soulmods_x64.dll" : "soulmods_x86.dll") is Dictionary<string, long>  exports)
```

Comment:
ER always 64 bit, we never need soulmods_x86.dll here. Maybe the soulmods inject() method should return the exported functions if the injections succeed, "soulmods_x64.dll" and "soulmods_x86.dll" are kind of magic strings that I already missed once in https://github.com/FrankvdStam/SoulSplitter/pull/73, so it would be good if we can contain them to the soulmods class.